core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -139,6 +139,64 @@ describe('Topic editor functionality', function() {
     });
   });
 
+  it(
+    'should edit the contents of the created question', function() {
+      topicEditorPage.get(topicId);
+      topicEditorPage.moveToQuestionsTab();
+      topicEditorPage.expectNumberOfQuestionsForSkillWithDescriptionToBe(
+        1, 'Skill 1');
+      // Get the questions list and edit the contents of the question.
+      element.all(
+        by.css('.protractor-test-question-list-item')).then(
+        function(questionItems) {
+          questionItems[0].click();
+          explorationEditorMainTab.setContent(forms.toRichText('Question 2'));
+          topicEditorPage.saveQuestion();
+          // Confirm the edit by adding a commit message.
+          topicEditorPage.saveCommitMessage('Edited the question content.');
+        });
+      topicEditorPage.get(topicId);
+      topicEditorPage.moveToQuestionsTab();
+      topicEditorPage.expectNumberOfQuestionsForSkillWithDescriptionToBe(
+        1, 'Skill 1');
+      // Confirm that the content of the question has changed.
+      element.all(
+        by.css('.protractor-test-question-list-item')).then(
+        function(questionItems) {
+          waitFor.visibilityOf(
+            questionItems[0], 'Question takes too long to appear');
+          questionItems[0].click();
+          explorationEditorMainTab.expectContentToMatch(
+            forms.toRichText('Question 2'));
+        });
+    });
+
+  it(
+    'should create an additional skill in the topic and edit the skill' +
+    'linkages for the existing question', function() {
+      topicsAndSkillsDashboardPage.get();
+      topicsAndSkillsDashboardPage.createSkillWithDescriptionAndExplanation(
+        'Skill 2', 'Concept card explanation');
+
+      topicsAndSkillsDashboardPage.get();
+      topicsAndSkillsDashboardPage.navigateToUnusedSkillsTab();
+      topicsAndSkillsDashboardPage.assignSkillWithIndexToTopic(0, 0);
+
+      topicEditorPage.get(topicId);
+      topicEditorPage.moveToQuestionsTab();
+      topicEditorPage.expectNumberOfQuestionsForSkillWithIndexToBe(1, 0);
+
+      element.all(
+        by.css('.protractor-test-question-list-item')).then(
+        function(questionItems) {
+          questionItems[0].click();
```

Comment:
This is the newly added question, right? If so, can you add a comment specifying this because it's not obvious that new questions get added to the top